nilearn/conftest.py

```diff
@@ -14,7 +14,7 @@
 except ImportError:
     collect_ignore = ['plotting',
                       'reporting',
-                      ]
+                      'datasets/data/convert_templates.py']
```

Comment:
'datasets/data/convert_templates.py' is only excluded when the matplotlib import fails. it should be excluded in all cases -- add it to collect_ignore outside of this except block

Reply:
AFAICS, there's only `collect_ignore` available in this block...

Reply:
it is a global variable that needs to contain the convertion script. so:
```python
collect_ignore = ["datasets/data/convert_templates.py"]
try:
    import matplotlib  # noqa: F401
except ImportError:
    collect_ignore.extend(['plotting', 'reporting'])
    matplotlib = None
```